Design/Design.xsd

```diff
@@ -395,25 +395,37 @@ elementFormDefault="qualified">
     <complexType name="DeviceLogicMutex"></complexType>
 
     <complexType name="Method">
-    	<sequence>
-    		<element name="argument" type="tns:MethodArgument"
-    			minOccurs="0" maxOccurs="unbounded">
-    		</element>
-    		<element name="returnvalue" type="tns:MethodReturnValue" minOccurs="0" maxOccurs="unbounded"></element>
+        <sequence>
+            <element name="argument" type="tns:MethodArgument" minOccurs="0" maxOccurs="unbounded"></element>
+            <element name="returnvalue" type="tns:MethodReturnValue" minOccurs="0" maxOccurs="unbounded"></element>
             <element name="documentation" type="tns:Documentation" maxOccurs="1" minOccurs="0"></element>
-    	</sequence>
-    	<attribute name="name" type="tns:VariableName" use="required"></attribute>
+        </sequence>
+        <attribute name="name" type="tns:VariableName" use="required"></attribute>
+        <attribute name="executionContext" type="tns:MethodExecutionContext" use="required"></attribute>
```

Comment:
Suggestion: how about calling this 'executionSynchronicity'? It might help to make it more apparent to the casual reader that this attribute concerns only the synchronous-ness of the method (my opinion: context is a rather vague term).

Reply:
OK! And since you are the only native speaker among us, it makes double sense ;-)

Have you got any other thoughts? Or that's the only one?

Reply:
I didn't see anything else. I do find XSLT pretty hard to parse though. I'll try and make a quick test/demo of the functionality for myself tomorrow. I'll post again after that.